app/blockchain/fabric-samples/test-network/addOrg3/README.md

```
@@ -0,0 +1,28 @@
+## Adding Org3 to the test network
+
+You can use the `addOrg3.sh` script to add another organization to the Fabric test network. The `addOrg3.sh` script generates the Org3 crypto material, creates an Org3 organization definition, and adds Org3 to a channel on the test network.
+
+You first need to run `./network.sh up createChannel` in the `test-network` directory before you can run the `addOrg3.sh` script.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)